src/status_im2/contexts/wallet/create_account/view.cljs

```diff
@@ -1,16 +1,78 @@
 (ns status-im2.contexts.wallet.create-account.view
-  (:require [react-native.core :as rn]
-            [quo2.core :as quo]
-            [re-frame.core :as rf]))
+  (:require
+    [quo2.foundations.colors :as colors]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.core :as quo]
+    [re-frame.core :as rf]
+    [react-native.safe-area :as safe-area]
+    [status-im2.contexts.wallet.common.temp :as temp]
+    [utils.i18n :as i18n]
+    [status-im2.contexts.wallet.create-account.style :as style]))
 
-(defn view
-  []
-  [rn/view
-   {:style {:flex            1
-            :align-items     :center
-            :justify-content :center}}
+(defn- view-internal
+  [{:keys [theme]}]
+  (let [top    (safe-area/get-top)
+        bottom (safe-area/get-bottom)]
```

Comment:
Since it's creating account flow, can we add the state(ratom) for the customization-color?

Reply:
@smohamedjavid fixed